diff_pdf_visually/diff.py

```diff
@@ -111,7 +110,7 @@ def pdfdiff_pages(
 
     Keyword arguments:
 
-    tempdir: if not None, then this should be a pathlib.Path for an empty writable
+    tempdir: if not None, then this should be a str or a pathlib.Path for an empty writable
```

Comment:
My preference is to keep the interface strict as it is: always a `Path` instead of something that's convertible to path. It reduces the number of ways that the code needs to be tested (or that it can break), and it doesn't seem a huge burden for the caller.

Reply:
As you prefer, it was just a proposal.